docs/architecture/adr-015-namespace-id-size.md

```diff
@@ -145,6 +145,31 @@ Another tradeoff to consider is the size of the namespace ID in the share. Since
 
 If the namespace ID size is increased, the maximum possible blob will decrease. Given the maximum possible blob is bounded by the number of bytes available for blob space in a data square, if a 32 byte namespace ID size is adopted, the maxmimum blob size will decrease by an upper bound of `appconsts.DefaultMaxSquareSize * appconsts.DefaultMaxSquareSize * (32-8)`. Note this is an upper bound because not all shares in the data square can be used for blob data (i.e. at least one share must contain the associated PayForBlob transaction).
 
+### SHA256 Performance
+
+If the namespace size is increased, whenever a SHA256 invocation takes place (e.g. NMT's [HashNode](https://github.com/celestiaorg/nmt/blob/fd00c52175c48bad64d03444689162fb9c6bee41/hasher.go#L265)), more data needs to be SHA256'ed. For example:
+
+Namespace size (bytes) |[HashNode](https://github.com/celestiaorg/nmt/blob/fd00c52175c48bad64d03444689162fb9c6bee41/hasher.go#L302)'s SHA256 max data (bytes)
```

Comment:
`Namespace size (bytes)` here does not seem to mean the same thing as `Namespace size (bytes)` in the earlier tables right? In this table it seems to include the version byte, whereas in the earlier tables they don't.

Reply:
Good point. This ADR was written before the namespace version was introduced. I'll clarify in all the tables